src/index.html

```diff
@@ -22,6 +22,355 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="Company logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              apple
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              samsung
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              tablets
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              photo
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <div class="card__image"></div>
+
+          <h2 class="card__apple">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
The product title is repeated multiple times across different cards. Consider using a dynamic approach to populate these fields to avoid repetition and potential errors in content updates.